distributed/cli/dask_scheduler.py

```diff
@@ -11,7 +11,7 @@
 
 import click
 
-import distributed
```

Comment:
Thank you for cleaning up.  We've been slowly extending flake8 testing throughout the codebase but haven't gotten here yet.